src/mlagility/analysis/status.py

```diff
@@ -133,6 +142,8 @@ def print_invocation(
 
     print(f"{ident}\tInput Shape:\t{input_shape}")
     print(f"{ident}\tHash:\t\t" + invocation_hash)
+    print(f"{ident}\tBuild Name:\t" + build_name)
+    print(f"{ident}\tCache dir:\t" + cache_dir + '/' + build_name)
```

Comment:
I think we are also overloading the term `cache_dir` here. Example:

```
Models discovered during profiling:

two_input_shapes.py:
        model
                Model Type:     Pytorch (torch.nn.Module)
                Class:          EfficientNet (<class 'timm.models.efficientnet.EfficientNet'>)
                Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/timm/models/efficientnet_builder.py, line 472
                Parameters:     1,677,128 (3.2 MB)

                With input shape 1 (executed 1x - 0.01s)
                Input Shape:    'Positional Arg 1': (1, 3, 28, 28)
                Hash:           4c4868c4
                Build Name:     two_input_shapes_timm_cbdf10c5
                Cache dir:      /home/dhnoronha/.cache/mlagility/two_input_shapes_timm_cbdf10c5


                With input shape 2 (executed 1x - 0.03s)
                Input Shape:    'Positional Arg 1': (1, 3, 224, 224)
                Hash:           cbdf10c5
                Build Name:     two_input_shapes_timm_cbdf10c5
                Cache dir:      /home/dhnoronha/.cache/mlagility/two_input_shapes_timm_cbdf10c5
```

Shouldn't `Cache dir` be  `/home/dhnoronha/.cache/mlagility/` instead of `/home/dhnoronha/.cache/mlagility/two_input_shapes_timm_cbdf10c5`?

Reply:
Adding the full cache dir path is intended to point the users to exactly where all the generated files are located. They can copy this path and either `cd` into it or use it elsewhere.

Reply:
Yes, I understand that. My comment is just about a nit in terminology.
`Cache Dir` is theoretically not the most correct terminology for that. I think the correct word for it, given what is on our code, is `Build Dir`.

```
build_dir = build.output_dir(cache_dir, build_name)
```
(from `filesystem.py, line 186`)

Reply:
We have been using "build dir" to describe the folder holding a build, and "cache dir" for the directory holding all the build dirs.

Reply:
Thats fair. I will rename `Cache dir` to `Build dir`. I am pointing to the build dir here